plugins/plugin-openwhisk/tests/lib/openwhisk/openwhisk.js

```diff
@@ -45,11 +45,11 @@ const localWskProps = () => {
 exports.entities = ['action', 'trigger', 'rule', 'package']
 
 const apihost = process.env.__OW_API_HOST || process.env.API_HOST || process.env.APIHOST || localWskProps().APIHOST
-const apihostIsLocal = apihost.indexOf('localhost') >= 0 ||
+const apihostIsLocal = apihost ? apihost.indexOf('localhost') >= 0 ||
       apihost.startsWith('192.') ||
       apihost.startsWith('172.') ||
       apihost.startsWith('https://192.') ||
-      apihost.startsWith('https://172.')
+      apihost.startsWith('https://172.') : null;
```

Comment:
I think we shouldn't change any logic in OpenWhisk tests for workaround. Instead, we should change the test logic on the higher layer, e.g. fail when users want to run openwhisk test, but no .wskprops exists; otherwise, we don't run openwhisk tests.